libs/schemas/src/entities/notification.schemas.ts

```diff
@@ -70,7 +70,8 @@ export const ThreadSubscription = z.object({
         }),
       }),
     )
-    .optional(),
+    .optional()
+    .nullish(),
```

Comment:
`optional()` is redundant if using `nullish()`. See https://gist.github.com/ciiqr/ee19e9ff3bb603f8c42b00f5ad8c551e.

Reply:
Removed